examples/templateservicebroker/apiserver-config.yaml

```diff
@@ -0,0 +1,4 @@
+kind: TemplateServiceBrokerConfig
+apiVersion: config.templateservicebroker.openshift.io/v1
+templateNamespaces:
+- openshift
```

Comment:
wouldn't the ansible installer be using the full TSB template file and customizing/deploying that?